jupytext/cli.py

```diff
@@ -594,7 +594,7 @@ def jupytext_single_file(nb_file, args, log):
             except NotAPairedNotebook as err:
                 sys.stderr.write("[jupytext] Warning: " + str(err) + "\n")
                 return 0
-            except (InconsistentVersions, InputNotInIndex) as err:
+            except (InconsistentVersions) as err:
```

Comment:
NIT:the parens are unnecessary now.

Reply:
Yes sure, I think I fixed that in a recent commit